### PR TITLE
Fix order of message of signaling server to be returned to client

### PR DESCRIPTION
**Pull request by vitormattos**
### 🚧 TODO

- [ ] Close https://github.com/nextcloud/spreed/issues/7863

### 🏁 Checklist

- [x] ⛑️ Tests (unit and/or integration) are included
- [x] 📘 API documentation in `docs/` has been updated or is not required
- [x] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required
- [x] 🔖 Capability is added or not needed

### CODE REVIEW

File: lib/Signaling/Messages.php [hunk 1]
```diff
@@ -129,7 +129,8 @@ public function getAndDeleteMessages(string $sessionId): array {
 		$query->select('*')
 			->from('talk_internalsignaling')
 			->where($query->expr()->eq('recipient', $query->createNamedParameter($sessionId)))
-			->andWhere($query->expr()->lte('timestamp', $query->createNamedParameter($time)));
+			->andWhere($query->expr()->lte('timestamp', $query->createNamedParameter($time)))
+			->orderBy('id');
```
Comment (nickvergessen) [inline lib/Signaling/Messages.php:133]:
```suggestion
			->orderBy('id', 'ASC');
```

Comment (vitormattos) [reply lib/Signaling/Messages.php:133]: ASC is the default to Doctrine DBAL:
https://github.com/nextcloud/3rdparty/blob/master/doctrine/dbal/src/Query/QueryBuilder.php#L1243
Follow as default at the latest version of DBAL:
https://github.com/doctrine/dbal/blob/3.6.x/src/Query/QueryBuilder.php#L1277

And also to
MySQL:
https://dev.mysql.com/doc/refman/8.0/en/sorting-rows.html
PostgreSQL:
https://www.postgresql.org/docs/current/queries-order.html
SQLite:
https://www.sqlitetutorial.net/sqlite-order-by/
And Oracle:
https://www.oracletutorial.com/oracle-basics/oracle-order-by/

Comment (nickvergessen) [reply lib/Signaling/Messages.php:133]: Explicit is better :)

Comment (vitormattos) [reply lib/Signaling/Messages.php]: https://github.com/nextcloud/spreed/issues/9077

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**